apps/remix-ide-e2e/src/tests/generalSettings.test.ts

```diff
@@ -18,13 +18,18 @@ module.exports = {
   'Should activate `generate contract metadata`': function (browser) {
     browser.waitForElementVisible('*[data-id="remixIdeSidePanel"]', 5000)
       .waitForElementVisible('*[data-id="settingsTabGenerateContractMetadataLabel"]', 5000)
+      .execute(function () {
+        const elem = document.getElementById('generatecontractmetadata') as HTMLInputElement
+        elem.checked = true
+      })
+      .verify.elementPresent('[data-id="settingsTabGenerateContractMetadata"]:checked')
       .click('*[data-id="verticalIconsFileExplorerIcons"]')
       .click('[data-id="treeViewLitreeViewItemcontracts"]')
       .openFile('contracts/3_Ballot.sol')
       .click('*[data-id="verticalIconsKindsolidity"]')
       .pause(2000)
       .click('*[data-id="compilerContainerCompileBtn"]')
-      .pause(3000)
+      .pause(30000)
```

Comment:
This waiting time is too long. 3000 is sufficient, maybe 5000 also.

Reply:
reduce it to 3000, i agree it was actually too long